src/web/mod.rs

```diff
@@ -351,7 +355,11 @@ fn match_version(
     {
         return Ok(MatchVersion {
             corrected_name,
-            version: MatchSemver::Semver((version.to_string(), *id)),
+            version: if input_version == Some("latest") {
```

Comment:
I thought what we discussed above was that `*` and `newest` should continue to have their current behavior. That's why I specifically went for `input_version` rather than `req_version` (which gets set to `*` for all three cases).

Reply:
My impression was they should redirect to *latest*, not to the exact version. It seems weird to have that be inconsistent.

(But uhhh yeah I did forget they should redirect, oops)

Reply:
In what context are `*` and `newest` exposed? Part of why I wanted to leave these as-is is that I don't fully understand the use case for these alternate names. Are they a fully supported feature? How do people get to them?

I _think_ supporting these by redirecting to `latest` will require an additional clause in each of the places we redirect, so I want to make sure this is something we want to do before going ahead with it.

Reply:
Yes, they are fully supported in that we document them on https://docs.rs/about/redirections.

Reply:
I looked at fixing this and it complicates the code somewhat. We'd have to add a _fourth_ MatchSemver variant, RedirectToLatest or similar. I propose to let `*` and `newest` continue to redirect to an exact version. It seems slightly weird, but it has the advantage of continuing the current behavior and not overcomplicating the code (and being less work :-D).

Reply:
Hmm, ok - it doesn't seem like the worst thing to have a way to opt-in to the old behavior :)